src/App.css

```diff
@@ -1,4 +1,5 @@
 .App {
+  margin-top: 62.5px;
```

Comment:
Why not just 60? 62.5 seems too precise.

Reply:
That’s the height of the top navigation header. The w3 schools CSS uses a font size of 15 and a line height of 1.5 so we a half pixel. I can just set this to 60. Being precise about the intent and implementation of the layout is my job ;¬)—but if the intent here is imprecise, well then I will take a look at what the original layout was doing exactly and match that.

Reply:
I double checked… `<div class="oct-head-margin">&nbsp;</div>` is 22.5px tall with `margin-top: 40px`, which is where I got 62.5 so that’s replicating exactly the offset as it exists today. Let me know if that doesn’t work for you, and I can round it down to 60 or whatever.

Reply:
It's fine either way.  The intent was so the scrolling nav-bar is visible under the non-scrolling top-level nav-bar.